### PR TITLE
French translation improvements from Crowdin made by:

### PR DESCRIPTION
**Pull request by magul**
* Brice Copy (https://crowdin.com/profile/bcopy)
* Loïc Giraudel (https://crowdin.com/profile/l_giraudel)

### CODE REVIEW

File: fr/chromebook_setup/instructions.md [hunk 1]
```diff
@@ -126,7 +126,7 @@ LOGGING = {
 
 Cela va créer un fichier `debug.log` détaillant les opérations Django et tous les messages d'erreur qui pourraient apparaître, ce qui rendra la correction beaucoup plus facile si votre site Web ne fonctionne pas.
 
-Le redémarrage initial du projet Glitch devrait échouer. (Si vous cliquez sur le bouton en haut de la liste déroulante `Show` puis cliquez sur `In a New Window`, vous recevrez un message d'erreur `DisallowedHost` .) Ne vous inquiétez pas à ce stade, le tutoriel corrigera cela dès que vous aurez mis à jour les paramètres Django de votre projet dans le fichier `mysite/settings.py` .
+Le redémarrage initial du projet Glitch devrait échouer. (Si vous cliquez sur le bouton en haut de la liste déroulante `Show` puis cliquez sur `In a New Window`, vous recevrez un message d'erreur `DisallowedHost` d'accès non autorisé .) Ne vous inquiétez pas à ce stade, le tutoriel corrigera cela dès que vous aurez mis à jour les paramètres Django de votre projet dans le fichier `mysite/settings.py` .
```
Comment (ariasuni) [inline fr/chromebook_setup/instructions.md:129]: I would have put «accès non autorisé» between parentheses. One wouldn’t say «une erreur 404 de fichier non trouvé».

**Review by ariasuni — CHANGES_REQUESTED** [review]
Looks good to me otherwise.

File: fr/chromebook_setup/instructions.md (hunk 1 above)
Comment (magul) [reply fr/chromebook_setup/instructions.md:129]: Can you, please, apply that change here: https://crowdin.com/project/django-girls-tutorial/fr# ?

I'm merely a steward trying to sync translations between Crowdin na Github.

Comment (ariasuni) [reply fr/chromebook_setup/instructions.md:129]: I updated the translation on Crowdin!

**Comment by magul** [comment]
@ariasuni refreshed from Crowdin, can you re-review?

**Review by ariasuni — APPROVED** [review]

**Pull request merged**